packages/ember-intl/addon/services/intl.js

```diff
@@ -89,6 +89,15 @@ const IntlService = Service.extend(Evented, {
 
     return this.formatMessage(translation, ...args);
   },
+  
+  
+  tH(key, ...args) {
```

Comment:
Well making it argument will be confusing with translations with variables. If HTML is filtered via arguments it will increase logic on ember-intl end also bit difficult on component end. 

tH bothered me too but what is the shortest and closest to existing this.t() function?